src/core/reducers/addons.js

```diff
@@ -355,8 +363,8 @@ export default function addonsReducer(
         byID[`${addon.id}`] = addon;
 
         if (addon.slug) {
-          bySlug[addon.slug] = addon.id;
-          loadingBySlug[addon.slug] = false;
+          bySlug[addon.slug.toLowerCase()] = addon.id;
```

Comment:
Should you also be using `slug.toLowerCase()` in `FETCH_ADDON` (line 346)?

Reply:
oh right, good catch!